scIB/metrics/metrics.py

```diff
@@ -149,6 +148,14 @@ def metrics_all(
         hvg_score_=True,
         graph_conn_=True,
         pcr_=True,
+        isolated_labels_f1_=True,
+        trajectory_=True,
+        nmi_=True,
+        ari_=True,
+        cell_cycle_=True,
+        kBET_=True,
+        ilisi_=True,
+        clisi_=True,
         **kwargs
     )
```

Comment:
Could add a deprecation comment for this as well... we don't use it anymore as it only works for python methods.

Reply:
Deprecation for the LISI parameters?